docs/jwst/jump/arguments.rst

```diff
@@ -38,23 +38,22 @@ The ``jump`` step has five optional arguments that can be set by the user:
   not have its neighbors flagged. The goal is to prevent flagging jumps that would be too
   small to significantly affect the slope determination.  The default value is 10.
 
-*  ``--after_jump_flag_dn1``: A floating point value in units of DN that gives the
-   minimum ramp jump for after jump flagging for the 1st after ramp flagging.
+  After a jump of at least 'after_jump_flag_dn1' DN, frames up to 'after_jump_flag_time1'
+  seconds will be also flagged as jumps. That pair of arguments are defined as:
```

Comment:
Need matching single/plural usage: "That pair of arguments *is* defined as ..."  (it's a single pair)

Reply:
Right